composer/algorithms/swa/README.md

```diff
@@ -0,0 +1,80 @@
+# 🧩 Stochastic Weight Averaging
+
+[\[How to Use\]](#how-to-use) - [\[Suggested Hyperparameters\]](#suggested-hyperparameters) - [\[Attribution\]](#attribution)
+
+ `Computer Vision`, `Natural Language Processing`
+
+Stochastic Weight Averaging (SWA) maintains a running average of the weights towards the end of training. This leads to better generalization than conventional training.
+
+| ![SWA](https://storage.googleapis.com/docs.mosaicml.com/images/methods/swa.png) |
+|:--|
+|*Visualization of SWA from an extensive [PyTorch blogpost about SWA](https://pytorch.org/blog/stochastic-weight-averaging-in-pytorch/).*|
+
+
+## How to Use
+
+### Composer Trainer
+
+<!--pytest.mark.gpu-->
+<!--pytest.mark.timeout(20)-->
+<!--
+```python
+from torch.utils.data import DataLoader
+from tests.common import RandomClassificationDataset, SimpleModel
+model = SimpleModel()
+train_dataloader = DataLoader(RandomClassificationDataset())
+eval_dataloader = DataLoader(RandomClassificationDataset())
+```
+-->
+<!--pytest-codeblocks:cont-->
+```python
+# Instantiate the algorithm and pass it into the Trainer
+# The trainer will automatically run it at the appropriate points in the training loop
+
+from composer.algorithms import SWA
+from composer.trainer import Trainer
+
+swa_algorithm = SWA(
+    swa_start="1ep",
+    swa_end="2ep",
+    update_interval='5ba',
+)
+trainer = Trainer(
+    model=model,
+    train_dataloader=train_dataloader,
+    eval_dataloader=eval_dataloader,
+    max_duration="3ep",
```

Comment:
Nit: can we make 2ep? Slightly faster and consistent with other tests

Reply:
Due to the nature of the algorithm, it doesn't make sense to run it for less than 3 epochs. I know this is an example, but I think the example should reflect the best practices of the method. Can go into detail if you're curious, but happy to change if necessary.